src/styles/blocks/card-grid.scss

```diff
@@ -0,0 +1,101 @@
+@import "../utils/variables";
+@import "../utils/functions";
+@import "../utils/mixins";
+@import "../utils/extends";
+
+.page__catalog {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.product-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: gridSize($grid-item-width, $grid-num-of-item,
+  $grid-margin-btw-item);
+
+  &__card:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  @extend %border-style;
+  border-color: #f3f3f3;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    @include font--main;
+    line-height: 18px;
+    color: $text-color_main-accent;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    color: $text-color__secondary-color;
+  }
+
+  &__reviews {
+    @extend %flex-style;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-ammount {
+    color: $text-color__secondary-color;
+    line-height: 16px;
+  }
+
+  &__price-block {
+    @extend %flex-style;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color__secondary-color;
+  }
+
+  &__price-ammount {
+    @include font--price;
+    color: $text-color_main-accent;
+  }
+
+  &__button-buy {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
```

Comment:
2 points:
- do not set precise width, just make it spread all over available space
- do not set precise height, let's use vertical paddings